app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(list_of_players: list) -> int:
+    return sum([player.get_rating() for player in list_of_players])
+
+
+def elves_concert(list_of_elfs: list) -> None:
```

Comment:
I don't think it's a reason to request changes, but you have spelling issues in **elfs** and **dwarfes**: there are examples of right spellings in the names of the functions. :)